### DISCUSSION TITLE
What permission level do I need to create issues using PAT

### DISCUSSION THREAD

**Discussion by tzaitoun** [community/community#24499] [Apps, API and Webhooks]
<p>I generated a personal access token and I want to give it the minimum permissions possible to be able to create issues. I gave it the <a href="https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/" rel="nofollow noopener">write:repo_hook</a> permission only, but I get “Not Found” when making the request. According to <a href="https://docs.github.com/en/rest/reference/issues#create-an-issue" rel="nofollow noopener">this</a>, anyone with pull access can create issues, I think this means I should also be able to set the permission to read instead of write, but the write doesn’t seem to work. I would appreciate any help, thanks.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>Thanks for joining the community. We’re glad you’re here!</p>
<p>The <code>*:repo_hook</code> permissions are for reading, writing, or administrating the webhooks associated with a repository. <a href="https://docs.github.com/en/developers/webhooks-and-events/about-webhooks">Webhooks</a> are a system that GitHub uses to notify external services of events on GitHub. So, <code>write:repo_hook</code> only gives you access to the webhooks for repositories and not access to the repository information itself.</p>
<p>For creating issues, you need either the <code>public_repo</code> or <code>repo</code> permission on your Personal Access Token. <code>public_repo</code> will only grant the ability to manipulate public repositories. <code>repo</code> will grant the ability to manipulate all repositories you have access to.</p>
<p>I hope that helps!</p>
